runtime/compiler/env/CHTable.cpp

```diff
@@ -1015,10 +1165,23 @@ TR_ClassQueries::collectImplementorsCapped(TR_PersistentClassInfo *clazz,
    {
    if (comp->getOption(TR_DisableCHOpts))
       return maxCount+1; // fail the collection
-   CollectImplementors collector(comp, clazz->getClassId(), implArray, maxCount, callerMethod, slotOrIndex, useGetResolvedInterfaceMethod);
-   collector.visitSubclass(clazz);
-   collector.visit(clazz->getClassId(), locked);
-   return collector.getCount(); // return the number of implementers in the implArray
+
+#if defined(J9VM_OPT_JITSERVER)
+   if (comp->isOutOfProcessCompilation())
+      {
+      CollectResolvedImplementors collector(comp, clazz->getClassId(), implArray, maxCount, callerMethod, slotOrIndex, useGetResolvedInterfaceMethod);
+      collector.visit(clazz->getClassId(), locked);
```

Comment:
Don't we need `visitSubclass(clazz)` like in the original? This visits the top class.

Reply:
Oh, true. I didn't realize that we don't visit the top class otherwise.